plugins/system_upgrade.py

```diff
@@ -451,6 +457,9 @@ def pre_configure_clean(self):
 
     def configure_download(self):
         if 'system-upgrade' == self.opts.command or 'fedup' == self.opts.command:
+            if is_rhel_system_present():
```

Comment:
We can get rid of the function and just use `if distro.id() == 'rhel':` here.

Reply:
Yep, sure. Hopefully there are no more `RHEL` conditions in the future :)